test-e2e/project-crud.js

```diff
@@ -124,6 +124,62 @@ test('CRUD operations', async (t) => {
         'expected values returns from getMany()'
       )
     })
+    t.test('create, close and then create, update', async (st) => {
+      const projectId = await manager.createProject()
+      const project = await manager.getProject(projectId)
+      const values = new Array(5).fill(null).map(() => {
+        return getUpdateFixture(value)
+      })
+      for (const value of values) {
+        // @ts-ignore
+        await project[schemaName].create(value)
+      }
+      // @ts-ignore
+      const written = await project[schemaName].create(value)
+      await project.close()
+
+      await st.exception(async () => {
+        const updateValue = getUpdateFixture(value)
+        // @ts-ignore
+        await project[schemaName].update(written.versionId, updateValue)
+      }, 'should fail updating since the project is already closed')
+
+      await st.exception(async () => {
+        for (const value of values) {
+          // @ts-ignore
+          await project[schemaName].create(value)
+        }
+      }, 'should fail creating since the project is already closed')
+
+      // @ts-ignore
+      await st.exception.all(async () => {
+        await project[schemaName].getMany()
+      }, 'should fail getting since the project is already closed')
+    })
+    t.test(
+      'create project, close, then re-create a project and create',
+      async (st) => {
+        // create project
+        const projectId = await manager.createProject()
+        const project = await manager.getProject(projectId)
+        // close it
+        await project.close()
+        // re create project
+        const newProjectId = await manager.createProject()
+        const newProject = await manager.getProject(newProjectId)
+        const newValues = new Array(5).fill(null).map(() => {
+          return getUpdateFixture(value)
+        })
+
```

Comment:
This isn't really testing anything - it's closing one project and creating a different one. Is this meant to test closing then re-opening a project? Then it should use the same projectId and check the data that was written before close is still there.

Reply:
Yeah, I realize this was the purpose of this tests. But I'm wondering how to achieve that. 
If I close a project I'm also closing the storage, and there's not an 'project.open()' or smth similiar. For now I've commented that test...

Reply:
@tomasciccola all of the storage instances will be re-created when you call `manager.getProject(...)`, but one thing that needs to be done is removing the cached project instances from the manager (`#activeProjects`) after closing the project. I have working changes that update the MapeoProject to emit a `'close'` event that we use to do this when creating the project instances in the MapeoManager. can push those changes with an updated working test if that's helpful

Reply:
see https://github.com/digidem/mapeo-core-next/pull/375/commits/5fee8466fa877445ec5727e957aeb215956d83b4 for relevant change